doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
+   content.  If you'd like to use this build of Chapel in a different
+   shell / terminal session, see :ref:`using-chapel-in-another-shell`
+   below.  Once you are comfortable with Chapel and interested in
    using a full-featured version in the preferred configuration, see
    the next section.
 
 
 .. _using-a-more-full-featured-chapel:
 
-Using a More Full-Featured Chapel
----------------------------------
+Using Chapel in its Preferred Configuration
+-------------------------------------------
 
-To use Chapel in a more full-featured and preferred configuration,
-you will need to rebuild Chapel from source in a different configuration.
-
-*  Open up a new shell to avoid inheriting the previous environment
-   settings.
-
-*  The quickstart configuration described above sets ``CHPL_LLVM=none``
-   but this is not the preferred configuration of Chapel. As of 1.25,
-   LLVM is the default compiler backend and it needs to be available
-   for full functionality. So make sure you have a compatible system LLVM
-   installed in a standard location or set the environment variable
-   ``CHPL_LLVM`` to ``bundled`` or ``none`` as described in
-   :ref:`chplenv.rst <readme-chplenv.CHPL_LLVM>`.  If desired, you must
-   explicitly opt-out of using it by setting
-   ``CHPL_LLVM=none``. The ``CHPL_LLVM=bundled`` build takes a long
-   time, so to avoid surprises you need to explicitly opt-in to using it.
-
-*  Repeat steps 2-5 above, but in Step 2, source ``util/setchplenv.bash``
-   instead of ``util/quickstart/setchplenv.bash``.
-   This will set up your environment to use Chapel in the preferred
-   configuration.  Building this configuration involves compiling
-   third-party packages, which will increase the overall build time.
-   If you run into any portability issues, please let us know via
-   :ref:`bugs.rst <readme-bugs>`.
+To use Chapel in its preferred, full-featured mode, you will need to
+rebuild Chapel from source in a different configuration:
 
-   .. code-block:: bash
+* Open up a new shell to avoid inheriting the previous environment
+  settings.
 
-      # Set environment variables to preferred configuration
-      source util/setchplenv.bash
+* The Quickstart configuration described above sets ``CHPL_LLVM=none``
+  for simplicity and to save time.  This causes the Chapel compiler to
+  use its C back-end, which is not the preferred option; as of Chapel
+  1.25, LLVM is the default back-end, which needs to be available for
+  full functionality.  There are a few options for using LLVM:
 
-      # re-build Chapel
-      make
+  - ensure that you have a version of LLVM 11 installed on your system
+    and set ``CHPL_LLVM=system`` (or leave it unset and Chapel should
+    find it if it's in your path)
 
-      # make check is available but optional
-      make check
+  - set ``CHPL_LLVM=bundled`` to have Chapel build and use the bundled
+    version of LLVM (note that building the bundled version of LLVM
+    can take a *long* time)
```

Comment:
I'm requesting that we take away the emphasis on long here.

I don't think that the LLVM build takes a *long* time but it does take a long time.

Reply:
How long do you find it takes?  How long is _long_?  I'm not sure I've had a build complete in less than an hour, possibly multiple hours?

Reply:
Right, it usually takes me 15 minutes, which is long but not *long*. I think a day would be *long* and while I respect everybody's choice to use or not use parallel builds I think *long* here is pessimistic (at least if we expect the reader might use a parallel build).

Reply:
OK, I definitely agree with removing the emphasis for a 15-minute build.  But... what's your secret?  Even with a parallel make, I don't believe I've gotten under an hour yet (admittedly, it was on chapcs rather than my Mac, which I admittedly became even more gun-shy about after #11404).

Reply:
The filesystem speed really matters for such things & I have a decent SSD

Reply:
Go on... So do you use `make -j` or `make -j <n>` and if so, with what value of n?  And are we talking about on a Mac, or another system?

Reply:
It's my home PC (running Ubuntu). I tend to use `make -j6` because with more parallelism I run out of memory. (I have 32GB). The system has 8 cores / 16 threads. 15 minutes is an approximate amount, not a measured number.